### PR TITLE
feat(isBlob): add `isBlob` function

### PR DESCRIPTION
**Pull request by lukaszkowalik2**
Hey,
I've added a new `isBlob` function.  If anyone is available to review the documentation in Korean or Japanese, I would greatly appreciate it!

### CODE REVIEW

**Comment by lukaszkowalik2** [comment]
I updated the test to mock the `File` object, as it’s a browser-only API, to prevent errors in CI environments where tests were failing. Does this look good?

File: docs/ja/reference/predicate/isBlob.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# isBlob
+
+指定された値がBlobであるかどうかをチェックします。
```
Comment (raon0211) [inline docs/ja/reference/predicate/isBlob.md:3]:
```suggestion
指定された値が`Blob`であるかどうかをチェックします。
```

File: docs/ja/reference/predicate/isBlob.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+function isBlob(x: unknown): x is Blob;
+```
+
+### パラメータ
+
+- `x` (`unknown`): Blobであるかどうかを確認する値。
+
+### 戻り値
+
+(`x is Blob`): 値がBlobであればtrue、そうでなければfalseを返します。
```
Comment (raon0211) [inline docs/ja/reference/predicate/isBlob.md:19]:
```suggestion
(`x is Blob`): 値がBlobであれば`true`、そうでなければ`false`。
```

File: docs/ko/reference/predicate/isBlob.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# isBlob
+
+주어진 값이 Blob인지 확인합니다.
```
Comment (raon0211) [inline docs/ko/reference/predicate/isBlob.md:3]:
```suggestion
주어진 값이 `Blob`인지 확인합니다.
```

File: docs/ko/reference/predicate/isBlob.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+이 함수는 TypeScript에서 타입 프레디케이트로 사용할 수 있으며, 인수의 타입을 `Blob`으로 좁힐 수 있습니다.
```
Comment (raon0211) [inline docs/ko/reference/predicate/isBlob.md:5]:
```suggestion
TypeScript의 타입 가드로 사용할 수 있어요. 파라미터로 주어진 값의 타입을 `Blob`으로 좁혀요.
```

File: docs/ko/reference/predicate/isBlob.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+function isBlob(x: unknown): x is Blob;
+```
+
+### 매개변수
+
+- `x` (`unknown`): Blob인지 확인할 값.
+
+### 반환값
+
+(`x is Blob`): 값이 Blob이면 true, 그렇지 않으면 false
```
Comment (raon0211) [inline docs/ko/reference/predicate/isBlob.md:19]:
```suggestion
(`x is Blob`): 값이 Blob이면 `true`, 그렇지 않으면 `false`.
```

**Review by raon0211 — APPROVED** [review]
Thanks for your contribution!

**Comment by raon0211** [comment]
> I updated the test to mock the File object, as it’s a browser-only API, to prevent errors in CI environments where tests were failing. Does this look good?

I guess this is the best way so far...

**Pull request merged**